src/client/pages/OfferNew/Perils/PerilModal.tsx

```diff
@@ -335,101 +171,47 @@ export const PerilModal: React.FC<PerilModalProps & ModalProps> = ({
     }, TRANSITION_MS * 2)
   }, [perils.length, currentPerilIndex, setCurrentPeril])
 
-  const tripledPerils = perils.concat(perils).concat(perils)
-
   const currentPeril = perils[currentPerilIndex % perils.length]
 
   return (
-    <Modal isVisible={isVisible} onClose={onClose}>
-      <Header>
-        <Picker>
-          <PerilItemsContainer
-            currentPerilIndex={currentPerilIndex}
-            totalNumberOfPerils={perils.length}
-            transition={transitionEnabled}
-          >
-            {tripledPerils.map((peril, index) => (
-              <PickerItem
-                key={index}
-                onClick={() =>
-                  actionsAllowed &&
-                  setCurrentPeril(index % tripledPerils.length)
-                }
-              >
-                <img
-                  src={peril.icon.variants.light.svgUrl}
-                  alt=""
-                  width={24}
-                  height={24}
-                />
-                <PickerItemLabel>{peril.title}</PickerItemLabel>
-              </PickerItem>
-            ))}
-          </PerilItemsContainer>
-          <Indicator />
-        </Picker>
-
-        <LeftGradient>
-          <BackButton
+    <ModalWrapper isVisible={isVisible} onClose={onClose}>
+      <ModalInnerWrapper>
+        <Header>
+          <DirectionButton
             onClick={() =>
               actionsAllowed && setCurrentPeril(currentPerilIndex - 1)
             }
           >
             <BackArrow />
-          </BackButton>
-        </LeftGradient>
-        <RightGradient>
-          <ForwardButton
+          </DirectionButton>
+          <Title>{currentPeril.title}</Title>
+
+          <DirectionButton
             onClick={() =>
               actionsAllowed && setCurrentPeril(currentPerilIndex + 1)
             }
           >
             <ForwardArrow />
-          </ForwardButton>
-        </RightGradient>
-      </Header>
-      <Content>
-        <Title>{currentPeril.title}</Title>
-        <Description>{currentPeril.description}</Description>
-
-        <CoverageWrapper>
-          <CoverageList>
-            <CoverageListTitle>
-              {textKeys.PERIL_MODAL_COVERAGE_TITLE()}
-            </CoverageListTitle>
-            {currentPeril.covered.map((text) => (
-              <CoverageListItem key={text}>
-                <Checkmark />
-                {text}
-              </CoverageListItem>
-            ))}
-          </CoverageList>
+          </DirectionButton>
+        </Header>
 
-          {currentPeril.exceptions.length > 0 && (
+        <Content>
+          <Description>{currentPeril.description}</Description>
+          <CoverageWrapper>
             <CoverageList>
               <CoverageListTitle>
                 {textKeys.PERIL_MODAL_EXCEPTIONS_TITLE()}
               </CoverageListTitle>
-              {currentPeril.exceptions.map((text) => (
+              {currentPeril.covered.map((text) => (
```

Comment:
This must be incorrect, the title is for exceptions but the list is what is covered, no?

Reply:
Should probably be `PERIL_MODAL_COVERAGE_TITLE`?

Reply:
We don't seem to have that title in Market web though 🤷

Reply:
Ok this explains a lot! Should I add them both so we have the covered items and the exceptions or just have what's covered like in market web?

Reply:
Ask MangZak